pkg/protostore/protostore.go

```diff
@@ -453,28 +452,24 @@ func (p ProtobufDataStore) DropTables(msgs ...proto.Message) error {
 }
 
 func (p ProtobufDataStore) MsgToPersist(ctx context.Context, id string, msg proto.Message, md Metadata) (ProtoStoreMsg, error) {
```

Comment:
@krishnamiriyala, could you add comments describing what this method does?

Reply:
done